code/modules/busy_space/organizations.dm

```diff
@@ -1078,12 +1078,12 @@
 	<br><br>\
 	SolGov defines top-level law (such as sapient rights and transgressive \
 	technology) and acts as an intermediary council for problems involving member states, but leaves most other law for \
-	states to define themselves. The member states of SolGov obey these laws, pay confederate taxes, and provide each \
+	states to define themselves. The member states of SolGov obey these laws, pay confederation taxes, and provide each \
 	other with military aid, in exchange for membership in the largest free trade, customs, and military union in the \
 	known galaxy. Each state appoints two representatives to the Colonial Assembly where issues are voted upon. \
```

Comment:
while you're already touching the file this is not accurate lol

Reply:
I think this is like First Draft Lore from 2016, the busy_space org blurbs are used literally nowhere so I'm hesistant to even put the effort in to revise it. I think they were intended to eventually fill a similar role to the WIP background system but are now dead and buried in code.

Reply:
i thought they auto populated the traveler's guide. if theyre actually not used anywhere at all then lol lmao who cares

Reply:
These can be overhauled or removed in a separate PR; it's not worth the effort to undo this edit as-is.